engine/src/main/scala/cromwell/webservice/CromwellApiHandler.scala

```diff
@@ -24,14 +24,7 @@ object CromwellApiHandler {
 
   final case class ApiHandlerWorkflowSubmit(source: WorkflowSourceFilesCollection) extends ApiHandlerMessage
   final case class ApiHandlerWorkflowSubmitBatch(sources: NonEmptyList[WorkflowSourceFilesCollection]) extends ApiHandlerMessage
-  final case class ApiHandlerWorkflowQuery(uri: Uri, parameters: Seq[(String, String)]) extends ApiHandlerMessage
```

Comment:
Is this all being removed because it was unused?

Asking because I just deleted this file so need to know if it's tied to any other changes.